container/lxd/initialisation.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/juju/utils/packaging/manager"
 	"github.com/juju/utils/proxy"
 
+	"syscall"
```

Comment:
Should we be using this package? It's deprecated, sort of, in favour of https://godoc.org/golang.org/x/sys/unix according to https://golang.org/pkg/syscall/

If we are using it it should be in the top block of imports.